apps/authentication/api/views.py

```diff
@@ -84,6 +84,28 @@ def dump_data(self, request, pk: int):
         return Response(data=serializer.data, status=status.HTTP_200_OK)
 
 
+class PermissionsViewSet(viewsets.ViewSet):
```

Comment:
Could be very useful for admins to see a users permissions as well.
Could therefore be useful to use a regular modelviewset which takes the user_id as a url param?

Reply:
Hm, you're right! Perhaps it should rather just be a subpart of `/users/{id}`? As in `/users/{id}/permissions`.

Reply:
Since we don't have any support for nesting with this project I think it works best as its own viewset, not under `/users`.  So `user/permissions/{user_id}/` I guess?

My guess is that you could use `user.get_user_permissions()` and turn that into a queryset of Permission somehow?

Reply:
> Or perhaps have both, so that you do not need to know your own ID to see your permissions? (although with OIDC, you have your ID from /userinfo)

Any logged in user should be able to get their id, so it would not be much of a problem.
The API-endpoints we have that do this without the user_id are probably ones I created before I knew any better

Reply:
Scratch that, I'm mixing things up...
`/users/{user_id}/permissions/` probably makes sense as an `@action` on the user. Some permission class like `IsSelfOrSuperUser`

For listing and editing permissions there should probably be a ModelViewSet.

Reply:
Made it a ReadOnlyModelViewSet for now, since editing requires additional logic which is not needed as of now.